target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -19,6 +19,9 @@ a60)
 	ucidef_set_led_default "status-red" "Status (red)" "a60:red:status" "0"
 	ucidef_set_led_default "status-blue" "Status (blue)" "a60:blue:status" "0"
 	;;
+ap531b0)
```

Comment:
Please, follow the general rule and keep all boards in alphabetical order in all scripts.